scripts/test/lint.py

```diff
@@ -354,6 +356,7 @@ def check(self, path):
 
 class UsesCategoryFromNuspec(Lint):
     # Some packages don't have a category (we don't create a link in the tools directory)
+    # we exclude dcode since it fails to install, issue tracked in #1175
```

Comment:
[nitpick] I think the comment should be in the same or previous line as the dcode exclusion. Otherwise I think it is likely that we forget to remove the comment when we remove the dcode exclusion
```suggestion
```